### PR TITLE
BlockSniper Version 1.5.0/2.0.0

### PR DESCRIPTION
**Pull request by Sandertv**
Pull for comparison only.

Do NOT merge yet. (There's probably multiple conflicts either way)

### CODE REVIEW

**Comment by TheDiamondYT1** [comment]
I think languages should be YAML to be easier for the end user to modify.

Yes, I know I'm the one that made them JSON but shhh 😉

**Comment by Sandertv** [comment]
Json is love, Json is life.

**Comment by TheDiamondYT1** [comment]
Very true

**Comment by SOF3** [comment]
YAML is a superset of JSON. You can safely change the parser (and the writer) to YAML and still safely load old JSON files.

File: src/BlockHorizons/BlockSniper/git/GitRepository.php [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+	public static function projectExists(int $projectId): bool {
+		return isset(self::$projects[$projectId]);
+	}
+
+	/**
+	 * @param int $projectId
+	 *
+	 * @return Project
+	 */
+	public static function getProject(int $projectId): Project {
```
Comment (TheDiamondYT1) [inline src/BlockHorizons/BlockSniper/git/GitRepository.php]: This needs to be nullable??

Comment (Sandertv) [reply src/BlockHorizons/BlockSniper/git/GitRepository.php]: Yeah I got that changed locally, I saw it too late.

**Pull request merged**